.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+#### Summary
+<!-- Provide a short summary of your changes -->
+
+#### Description
+<!-- Describe the changes in this PR here and provide some context -->
+
+#### Reviewers
+<!-- The persons assigned to review this PR should mark their item as done when they are happy with this PR to be merged -->
+
+- [ ] Reviewer 1 has approved the PR
+- [ ] Reviewer 2 has approved the PR
```

Comment:
We might get rid of this now that github supports this out of the box. We can remove it also from the boilerplate.

/cc @hisabimbola @PhilippSpo @Siilwyn